packages/ui/src/components/Designer/index.tsx

```diff
@@ -247,12 +253,7 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
           onEditEnd();
         }}
         zoomLevel={zoomLevel}
-        setZoomLevel={(zoom) => {
-          if (mainRef.current) {
-            mainRef.current.scrollTop = getPagesScrollTopByIndex(pageSizes, pageCursor, scale);
```

Comment:
I tried to align the left scroll centrally but this did not seem worth the effort.
Similarly, I didn't think pulling the viewport to the top was good UX, it felt OK to just zoom in / out from where you were.